Database/Corrections/sodQuestFixes.lua

```diff
@@ -2358,6 +2358,33 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -424992,
             [questKeys.zoneOrSort] = sortKeys.ROGUE,
         },
+        [90139] = {
+            [questKeys.name] = "Quick Strike",
+            [questKeys.startedBy] = {{1161,1162,1163,1164}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 10,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.WARRIOR,
+            [questKeys.objectivesText] = {"Kill Troggs for the Skull-Shaped Geode. Then get hit by a Skullthumper's stun and use the geode to loot the rune."},
+            [questKeys.requiredSpell] = -425443,
+            [questKeys.zoneOrSort] = sortKeys.WARRIOR,
+        },
+        [90140] = {
+            [questKeys.name] = "Quick Strike",
+            [questKeys.startedBy] = {
+                ['objectStart'] = {408799},
+                ['creatureStart'] = {504,95,449,590,589,121,285,46,732,735,210487},
+            },
```

Comment:
Sorry, but no, this is not correct 😅 

The `startedBy` table is number-indexed aka the order-matters. In `questDB.lua` we just give each of the indexes a name (e.g. `creatureStart`) to explain what each position is used for.

The correct format is:
```suggestion
            [questKeys.startedBy] = {{504,95,449,590,589,121,285,46,732,735,210487},{408799}},
```

The first entry (index=1) of `startedBy` is a table of NPC IDs (`creatureStart`), the second one is for object IDs (`objectStart`).

Reply:
https://github.com/Questie/Questie/pull/5435/commits/d454176f4acbdcf6ebf889e1814471a2a69266ef